src/prpy/planning/vectorfield.py

```diff
@@ -197,32 +219,197 @@ def TerminateMove():
 
             return Status.CONTINUE
 
+        integration_interval = 10.0
         return self.FollowVectorField(robot, vf_straightline, TerminateMove,
+                                      integration_interval,
+                                      timelimit, 
+                                      **kw_args)
+
+
+    @PlanningMethod
+    def PlanWorkspacePath(self, robot, traj,
+                          timelimit=5.0,
+                          position_tolerance=0.01,
+                          angular_tolerance=0.15,
+                          goal_pose_tol=0.01,
+                          Kp_ff=None,
+                          Kp_e=None,
+                          **kw_args):
+        """
+        Plan a configuration space path given a workspace path.
+        Trajectory timing information is ignored.
+
+        @param openravepy.Robot      robot: The robot.
+        @param openravepy.Trajectory traj:  Workspace trajectory,
+                                            represented as an
+                                            OpenRAVE AffineTrajectory.
+        @param float timelimit: Max planning time (seconds).
+        @param float position_tolerance: Constraint tolerance (meters).
+        @param float angular_tolerance:  Constraint tolerance (radians).
+        @param float goal_pose_tol: Goal tolerance (meters).
+        @param numpy.array Kp_ff: Feed-forward gain.
+                                  A 1x6 vector, where first 3 elements
+                                  affect the translational velocity,
+                                  the last 3 elements affect the
+                                  rotational velocity.
+        @param numpy.array Kp_e: Error gain.
+                                 A 1x6 vector, where first 3 elements
+                                 affect the translational velocity,
+                                 the last 3 elements affect the
+                                 rotational velocity.
+
+        @return openravepy.Trajectory qtraj: Configuration space path.
+        """
+
+        if not util.IsTrajectoryTypeIkParameterization(traj):
+            raise ValueError("Trajectory is not a workspace trajectory, it "
+                             "must have configuration specification of "
+                             "openravepy.IkParameterizationType.Transform6D")
+
+        #if IsTimedTrajectory(traj):
+        #    raise ValueError("PlanWorkspacePath expected an un-timed "
+        #                     "trajectory.")
+
+        if position_tolerance < 0.0:
+            raise ValueError('Position tolerance must be non-negative.')
+        elif angular_tolerance < 0.0:
+            raise ValueError('Angular tolerance must be non-negative.')
+        elif goal_pose_tol < 0.0:
+            raise ValueError('Goal-pose tolerance must be non-negative.')
+
+        #
+        #
+        # ADD ComputeGeodesicUnitTiming()
+        #
+        #
```

Comment:
I believe this is necessary for the function to work, since you no longer require that the input `path` is timed.